### DISCUSSION TITLE
6.12.5 does not work properly without a monitor

### DISCUSSION THREAD

**Discussion by KinShin** [devmfc/debian-on-amlogic#113] [General]
Thanks for your work.

I’m using your image on my HK1BOX (s905x3).

Everything worked fine until I installed the linux-image-6.12.5-meson64_20241218_arm64.deb.

I discovered that if I don’t connect my box to a monitor, I won’t be able to log in via SSH, and many services I’ve set up won’t run.

Now I’ve decided to buy an HDMI cheater to see if my box can work properly using the cheater.

I think others might be encountering the same problem, so I’m starting this discussion.

**Comment by devmfc** [comment]
Thanx for reporting this! This is a problem, I'm sorry. I am able to reproduce it and am looking for a solution. 

In the mean time you can just downgrade to the previous kernel by modifying the boot.config.

**Reply by KinShin** [reply]
So nice of you.

I've downgraded 6.6.60 following your guide.

I'm glad to test later build if you need.

Thanks again.

**Comment by devmfc** [comment]
I've found a temporary workaround: add the following line to /boot/boot.config (boot.config on first partition of disk):

`bootargs0=clk_ignore_unused`

It will boot now without a monitor attached.
I have not found the root cause yet, I'm afraid it will have to wait till after x-mas ;-)

**Reply by KinShin** [reply]
This issue isn’t so pressing now.
What really matters is enjoying a joyful Christmas.
Wishing you a Merry Christmas!

**Comment by KinShin** [comment]
[6.12.6](https://github.com/devmfc/debian-on-amlogic/releases/tag/v6.12.6) fixed this issue.

Thanks devmfc and close discussion.
